src/main/java/org/reaktivity/nukleus/http2/internal/Http2Connection.java

```diff
@@ -284,7 +284,7 @@ void handleData(
                 decodeError != null && decodeError != Http2ErrorCode.NO_ERROR)
         {
             // TODO: use traceId ??
-            http2Streams.forEach((i, s) -> s.onAbort(0));
+            http2Streams.forEach((i, s) -> s.onAbort(factory.supplyTrace.getAsLong()));
```

Comment:
Should this use the inbound `traceId` since `ABORT` is in the same logical direction and caused by the inbound `DATA` frame, right?

Please review other cases to make sure `traceId` is always propagated when reaction is in same direction as cause, i.e. `DATA => ABORT` or `ABORT => DATA`, etc.